packages/flutter/lib/src/painting/box_painter.dart

```diff
@@ -526,13 +526,18 @@ class BoxShadow {
   /// The amount the box should be inflated prior to applying the blur.
   final double spreadRadius;
 
+  /// Converts a blur radius in pixels to sigmas.
```

Comment:
You probably still want the `See the sigma argument to [MaskFilter.blur].` part.